.env.example

```diff
@@ -16,6 +16,4 @@ NODE_VERSION=X-Node-Version
 
 NODE_ENV = development
 
-
-
-
+NUMBERS_API_URL=http://numbersapi.com/random
```

Comment:
Good! But remember that is not good to put real data in this file,  especially sensible information like tokens, internal endpoints, passwords etc.